### ISSUE TITLE
HTML does not fit content, white gap on bottom and right of page

### ISSUE THREAD

**Issue by mgartner** [wkhtmltopdf/wkhtmltopdf#2126]
I'm creating a 6.25in x 4.25in PDF with no margins from HTML set to exactly the same size. Instead of fitting the entire rendered HTML into the first page of the PDF, it adds a small white gap on the bottom and the right of the page, and pushes the remaining content to the second page.

![screenshot 2015-01-02 17 17 14](https://cloud.githubusercontent.com/assets/1128750/5601140/a8a74594-92a3-11e4-94e4-6de4dd99d442.png)

I'm using wkhtmltopdf 0.12.1 on Mac OS X 10.10. Are there any workarounds for this bug?

**Comment by Jelle-S** [comment]
Same issue here:
wkhtmltopdf 0.12.1 on Mac OS X 10.10

**Comment by ashkulz** [comment]
Nope, could be rounding issue somewhere in Qt/WebKit but extremely hard to track down. Can you try if `0.13` alpha gives better results in a Windows VM?

**Comment by cerdiogenes** [comment]
+1

**Comment by Menu-Cast** [comment]
Hitting the same issue aswell, Have tried everything and have set all the dimensions correctly.

**Comment by Hyperblaster** [comment]
Same issue. Have tried everything but always have a tiny white border and it throws out all the pages.

**Comment by cerdiogenes** [comment]
@Hyperblaster, if you can downgrade wkhtmltopdf version there is a solution. I'm using 0.9.6, with linux, to avoid this bug with the following options:

dpi: 75,
margin: {
  top: 0, bottom: 0, left: 0, right: 0
},
disable_smart_shrinking: true

But if you need any behavior from the most recent versions I don't know any workaround.

**Comment by Godisemo** [comment]
+1

I currently have the same issue.

**Comment by apansky** [comment]
+1

**Comment by DominikPalo** [comment]
+1

**Comment by apansky** [comment]
btw, setting pdf dimensions in px and rendered divs in same dimensions + 1px sovled this for me.

**Comment by martin-trajanovski** [comment]
+1

**Comment by Tomsgu** [comment]
Is this issue persists in 0.12.5? Or does 0.13 gives a better results?
